src/main/java/javapoz24/team3/rental/domain/rental/RentalInfo.java

```diff
@@ -4,12 +4,28 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
 public class RentalInfo {
+
+    // Czy potrzebujemy tu id? Nasz rental ma zawsze id 1 - ale jeśli baza nada inny???
```

Comment:
Jeżeli prześlesz dane na widok które już istnieją w bazie a następnie je zapiszesz (zmodyfikujesz coś) nie podająć id poprawnego to hibernate potraktuje to jako nowy obiekt i skończysz z 2 obiektami zamiast jednym, zmodyfikowanym